inferno-core/src/Inferno/Eval.hs

```diff
@@ -169,6 +169,11 @@ eval env@(localEnv, pinnedEnv) expr = case expr of
         argv <- eval env arg
         f argv
       _ -> throwM $ RuntimeError "failed to match with a function"
+  -- TODO create pattern for LetAnnot, or reuse?
```

Comment:
yeah `LetAnnot_` might be nice

Reply:
I was wondering if there's a way to reuse the `Let_` pattern for both cases since the eval code is the same for either case as it ignores annotations?

Reply:
Don't think that's possible...